context/app/static/js/components/detailPage/DetailPageSection/CollapsibleDetailPageSection.tsx

```diff
@@ -72,12 +74,13 @@ export default function CollapsibleDetailPageSection({
               }}
               ml="auto"
               className="accordion-section-action"
+              sx={{ position: 'absolute', right: 0, top: 0 }}
```

Comment:
This is a creative workaround to the issue!

A minor nitpick: 
* the actions are very slightly misaligned relative to the accordion heading; could we adjust the `top` value accordingly?
  * Local: 
![image](https://github.com/user-attachments/assets/b1122fc8-390a-4d8d-a342-187eba1a132f)
  * Prod: 
![image](https://github.com/user-attachments/assets/ca849ba8-3a19-4a92-98e9-131b92e7b321)

Behaviorally it matches, so this should be just a minor `top` adjustment.

Reply:
Good catch! Should be aligned now:
![Screenshot 2024-12-20 at 12 09 11 PM](https://github.com/user-attachments/assets/0b1b5cd8-5433-43b3-832b-da8079b57b5f)